packages/backend-core/src/db/couch/DatabaseImpl.ts

```diff
@@ -83,7 +83,14 @@ export class DatabaseImpl implements Database {
       throw new Error("DB does not exist")
     }
     if (!exists) {
-      await this.nano().db.create(this.name)
+      try {
+        await this.nano().db.create(this.name)
+      } catch (err: any) {
+        // Handling race conditions
+        if (err.statusCode !== 412) {
```

Comment:
@Rory-Powell @mike12345567 thoughts on this?

Reply:
It was happening in multiple places, I actually needed retries because of this mainly. But the biggest conflicting one was `global-db`